dali/python/nvidia/dali/fn.py

```diff
@@ -67,7 +67,13 @@ def op_wrapper(*inputs, **arguments):
         def is_data_node(x):
             return isinstance(x, _DataNode)
         def is_call_arg(name, value):
-            return name == "name" or is_data_node(value)
+            if name == "name" or is_data_node(value):
+                return True
+            if value is None:
+                return False
+            if isinstance(value, (bool, int, float, str, list, tuple, nvidia.dali.types.ScalarConstant)):
```

Comment:
are this all the types we need to check? I am thinking of things like np.int64, etc

Reply:
Good point.